book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :name, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Dear @touredri, to make sure the association is correct we need to receive the `(person, date)` parameters and create the new rental inside the `add_rental` method by adding the `self` for the current book. With your current implementation, it's possible to use instance `X` to create a rental with an instance `Y`. 

Please use something like 
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```